buildenvs/base.Dockerfile

```diff
@@ -0,0 +1,146 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022, NEC Europe Ltd., Unikraft GmbH, and The KraftKit Authors.
+# Licensed under the BSD-3-Clause License (the "License").
+# You may not use this file except in compliance with the License.
+
+ARG REG=kraftkit.sh
+
+ARG UK_ARCH=x86_64
+
+ARG GCC_VERSION=12.2.0
+ARG GCC_SUFFIX=
+
+ARG QEMU_VERSION=7.1.0
+
+ARG KRAFTKIT_VERSION=latest
+
+ARG DEBIAN_VERSION=bullseye-20221114
+
+FROM ${REG}/gcc:${GCC_VERSION}-x86_64${GCC_SUFFIX} AS gcc-x86_64
+# FROM ${REG}/gcc:${GCC_VERSION}-arm${GCC_SUFFIX}    AS gcc-arm
+# FROM ${REG}/gcc:${GCC_VERSION}-arm64${GCC_SUFFIX}  AS gcc-arm64
+
+FROM ${REG}/qemu:${QEMU_VERSION}                   AS qemu
+FROM ${REG}/myself:${KRAFTKIT_VERSION}             AS kraft
+
+LABEL MAINTAINER="Alexander Jung <alexander.jung@unikraft.io>"
+LABEL MAINTAINER="Cezar Craciunoiu <cezar.craciunoiu@unikraft.io>"
```

Comment:
The `LABEL MAINTAINER` [deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated) and can be removed.